blueprints.yaml

```diff
@@ -20,12 +20,30 @@ form:
   validation: loose
   fields:
     enabled:
-      type: toggle
       label: PLUGIN_ADMIN.PLUGIN_STATUS
+      type: toggle
       highlight: 1
       default: 0
       options:
-        1: PLUGIN_ADMIN.ENABLED
         0: PLUGIN_ADMIN.DISABLED
+        1: PLUGIN_ADMIN.ENABLED
```

Comment:
Same question - why the change? I'd prefer seeing the default as the first option

Reply:
Because I like to have it in increasing order. But it's not really important for me. By the way: according to the file the default is 0.

Reply:
I'm not even sure why I had default as disabled 😕 It should've been enabled from the beginning